### ISSUE TITLE
D3D9 Port: Use Vertex Shader Model 2.0 or higher

### ISSUE THREAD

**Issue by LukeUsher** [Cxbx-Reloaded/Cxbx-Reloaded#1361]
This will allow the use of more instruction slots and registers, allowing Xbox vertex shaders to be more accurately converted to the host. 

This has potential to solve a massive amount of rendering issues, from broken polygons, missing animations, t-pose models to crashes.

As with Pixel Shaders, Shader Model 2.X is preferred over 2.0. 3.0 would be better, but is more work to implement.

**Note**:  This issue depends on the completion of #1359, which in-turn, depends on #1358 

Resources:
https://docs.microsoft.com/en-us/windows/desktop/direct3dhlsl/dx9-graphics-reference-asm-vs

**Comment by ObiKKa** [comment]
Is this new PR #1377 directly relevant to this milestone issue?

Also, thanks to all of the contributors here for helping to improve this emulator even while I was looking at the updates to many other emulators.

**Comment by LukeUsher** [comment]
Yeah, PR #1377 addresses this issue by moving to Shader Model 2.0 for Vertex Shaders

**Comment by PatrickvL** [comment]
Close this now?
